src/geocat/f2py/checks.py

```diff
@@ -1,6 +1,6 @@
-import xarray as xr
-import numpy as np
 import dask as da
+import numpy as np
+import xarray as xr
```

Comment:
What is reordering the imports?

Reply:
It is the pre-commit hook `isort` (i.e. "import sort"). See inside [.pre-commit-config.yaml](https://github.com/NCAR/geocat-f2py/blob/update_actions/.pre-commit-config.yaml) 

```
-   repo: https://github.com/PyCQA/isort    # To sort the imports
    rev: 5.7.0
    hooks:
    - id: isort
      exclude: src/geocat/f2py/fortran/
```

Reply:
thanks resolveing